src/components/news/article.tsx

```diff
@@ -55,13 +57,12 @@ const HeaderImageStyles = css`
     }
 `;
 
-const Article = ({ capi, imageSalt, children }: ArticleProps): JSX.Element => {
+const Article = ({ capi, imageSalt, article, children }: ArticleProps): JSX.Element => {
 
-    const { fields, tags, webPublicationDate, pillarId } = capi;
+    const { fields, tags, webPublicationDate } = capi;
     const series = articleSeries(capi);
-    const feature = isFeature(capi) || 'starRating' in fields;
-    const pillar = pillarFromString(pillarId);
-    const pillarStyles = getPillarStyles(pillar);
+    const feature = article.layout === Layout.Feature || article.layout === Layout.Review;
```

Comment:
Maybe another PR as it's not related but `feature` could be changed to `boldHeadline` as that's a bit more accurate of what it does?

Reply:
When we're describing an article of that type what term would we use?

Reply:
Oh no I'm referring to the `feature` prop which is basically just a boolean to show a bold headline.

Reply:
As discussed offline, I think I'm going to drop this prop in a follow-up.